### PR TITLE
50 update landing functionality

### LINKED ISSUE #50: Update landing page
- add rest of components into landing page
- fix theme
- add prop types to major components

### PR DESCRIPTION
**Pull request by LemonFace0309**
closes #50 

image

mostly added components to landing page but also adjusted theme and added propTypes to straggling pages. I didn't go through all components since many of them will be revised/deleted later anyway.

@eden-chan you can take a read too if you're interested :D

### CODE REVIEW

**Review by mabelene — COMMENTED** [review]
looks good :)) 
small edits to consider (not gonna put it as a task in case you want to just push this and make it an issue in the future):
- left side of the page needs padding
- big features need to be centred on the page

File: pages/landing/index.js [hunk 1]
```diff
@@ -74,6 +82,58 @@ export const getStaticProps = async () => {
...
+            'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pretium bibendum mauris, mollis...',
+        },
+        {
+          isPercentage: false,
+          number: 12,
+          description:
+            'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pretium bibendum mauris, mollis...',
+        },
+        {
+          isPercentage: true,
```
Comment (jimmyygit) [inline pages/landing/index.js:99]: is `isPercentage` necessary? Why cant we just put `99%` as text? I dont think all stats are just numbers or percentages. what if we want `100K` or `9/10 students...` This will probably change the prop types.

File: components/Landing/SmallFeatures/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  },
+}));
+
+const SmallFeatures = ({ features }) => {
+  const classes = useStyles();
+
+  return (
+    <>
+      <Typography variant="h3" align="center" gutterBottom>
+        And that&#39;s not all. We have more.
```
Comment (jimmyygit) [inline components/Landing/SmallFeatures/index.js:24]: what the rationale behind `&#39;`? Im pretty sure all devices nowadays know what an `'` apostrophe is. itll probably be easier to read in code too.

**Review by jimmyygit — APPROVED** [review]
lookies goodie toie meie

File: pages/landing/index.js (hunk 1 above)
Comment (LemonFace0309) [reply pages/landing/index.js]: Good point. I can dumb it down.

File: components/Landing/SmallFeatures/index.js (hunk 2 above)
Comment (LemonFace0309) [reply components/Landing/SmallFeatures/index.js:24]: eslint doesn't like that. It forces &#39; lol
